Remotion/Data/DomainObjects/Persistence/Rdbms/SqlServer/SchemaGeneration/SqlIndexDefinitionScriptElementFactory.cs

```diff
@@ -45,9 +45,10 @@ public override IScriptElement GetCreateElement (SqlIndexDefinition indexDefinit
          ownerName.EntityName,
          GetIndexedColumnNames(indexDefinition.Columns),
          indexDefinition.IncludedColumns != null
-             ? "\r\n  INCLUDE (" + string.Join(", ", indexDefinition.IncludedColumns.Select(c => "[" + c.Name + "]")) + ")"
+             ? $"{Environment.NewLine}  INCLUDE (" + string.Join(", ", indexDefinition.IncludedColumns.Select(c => "[" + c.Name + "]")) + ")"
```

Comment:
```suggestion
             ? Environment.NewLine + "  INCLUDE (" + string.Join(", ", indexDefinition.IncludedColumns.Select(c => "[" + c.Name + "]")) + ")"
```
Since there is already a ton of string contat going on, using string interpolation to prefix the newline feels very wrong

Reply:
Yeah there are some edge cases if you do everything with a macro :p

How about using interpolation everywhere instead?

```cs
$"{Environment.NewLine}  INCLUDE ({string.Join(", ", indexDefinition.IncludedColumns.Select(c => $"[{c.Name}]"))})"
```

I am not a fan of manual string concat as it makes it harder to read what whitespaces are actually part of the string and what are spacing because of the concat.

Reply:
's okay